Sources/Datasets/datasets/image/MNIST.swift

```diff
@@ -0,0 +1,25 @@
+struct MNISTBatch: S5TFLabeledBatch {
+    var data: Int
+    var labels: Int
+}
+
+struct MNIST: S5TFDataset {
+    typealias _Iterator = MNISTIterator // swiftlint:disable:this type_name
+    let data = [[1, 2], [3, 4], [5, 6]]
+    func batched(_ batchSize: Int) -> MNISTIterator {
+        return MNISTIterator(dataset: self)
+    }
+}
+
+struct MNISTIterator: S5TFDatasetIterator {
+    var dataset: MNIST
+
+    var count = 0
```

Comment:
Can we change this `count` variable name, it's a bit conflicting imo.

Reply:
Yes, you are right. I'll rename it to `index` and make it private.